styles.css

```diff
@@ -0,0 +1,111 @@
+body {
+  font-family: "Bebas Neue", sans-serif;
+  margin: auto;
+  font-size: 1.3rem;
+}
+
+h1 {
+  text-align: center;
+  font-size: 3rem;
+  background-color: #b1c5c3;
+  margin-top: 0;
+  padding: 10px;
+}
+
+.questionBox {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  margin-bottom: 30px;
+}
+
+.img-bookmark-white {
+  height: 40px;
+  position: absolute;
+  top: -18px;
+  right: 45px;
+}
+
+.questionCard {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  text-align: center;
+  flex-wrap: wrap;
+  align-items: left;
+  width: 70vw;
+  height: 25vh;
```

Comment:
please don't give fixed values for the width and height. Lets use the CSS in a way that the browser can decide how much space one card is needed. Please remove these values.